pom.xml

```diff
@@ -470,6 +494,7 @@
             <groupId>org.apache.hadoop</groupId>
             <artifactId>hadoop-mapreduce</artifactId>
           </exclusion>
+          <!--
```

Comment:
How are they runtime scoped in adam-assembly? They come in through a provided dependency...?